cms/cms-import.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Configuring Imports and Exports
+---
+
+[[toc]]
+
+Handling bulk data in Skygear CMS is easy. Your users can import data with a CSV and export data to a CSV. 
+
+## Imports
+
+![Skygear CMS import](/assets/cms/cms-import.png)
+
+Your users can import data to Skygear with a CSV. To do so, you need to configure the import settings and add an import button to the right page.
+
+### Configuring imports
+
+There are a few items you can configure in the import settings, and we are using the following 2 tables to illustrate how.
+
+**Blogpost**
+|Fields| Type| 
+|---|---|
+|_id| String|
+|_created_at| DateTime|
+|_updated_at| DateTime|
+|title| String|
+|content| String|
+|cover| Asset|
+
+**Comment**
+|Fields| Type| 
+|---|---|
+|_id| String|
+|_created_at| DateTime|
+|_updated_at| DateTime|
+|comment| String|
+|blogpostId | Reference|
+
+This is what we need:
+
+1. Allow users to import a list of comments. 
+2. We will use `comment` as an identifier. When there are two identical comments, they are considered as duplicated records.
+3. When there are duplicated records, we will throw an error to the users.
+4. To correctly associate every comment to a blogpost, we should also input the blogpost's title next to each comments.
+
+```yml
+site:
+  - type: Record
+    name: comment
+  - type: Record
+    name: blogpost
+
+imports:
+  import-comment:
+    record_type: comment
+    identifier: content
+    handle_duplicated_identifier: throw-error
+    fields:
+      - name: comment
+      - name: blogpostId
+        reference_target: blogpost
+        reference_field_name: title
+        handle_duplicated_reference: use-first
+
+records:
+  comment: 
+    record_type: comment
+  blogpost: 
+    record_type: blogpost
+```
+
+This is what your CSV should look like:
+
+|comment| title|
+|---| ---|
+|Can't agree more| Adele is so adorable!|
+|Let's go drinking this Friday! |Today is such a tired day|
+|Of course! |Skygear is so awesome|
+
+Configurable items:
+
+- <a href='./#record_type'>`record_type`</a>
+- <a href='./#identifier'>`identifier`</a>
+- <a href='./#handle_duplicated_identifier'>`handle_duplicated_identifier`</a>
+- <a href='./#fields'>`fields`</a>
+
+<a name='record_type'></a>
+
+#### `record_type`
+The database table this import is linked to.
+```yml
+imports:
+  import-name-random: 
+    record_type: 
+```
+
+<a name='identifier'></a>
+
+#### `identifier`
+The field that Skygear uses to determine if there are any duplicated records.
```

Comment:
Maybe we could mention if `identifier` default is `_id`, if it is not provided?

Reply:
Provided that in the CSV _id is provided?

Reply:
fixed.